package.json

```diff
@@ -18,6 +18,7 @@
     "csp:install": "cd libcsp &&  python3 waf install",
     "full_build": "yarn install_dependencies && yarn csp:clone && yarn build",
     "cli": "LD_LIBRARY_PATH=./libcsp/build PYTHONPATH=./libcsp/build python3 src/cli.py",
+    "cli_gdb": "LD_LIBRARY_PATH=./libcsp/build PYTHONPATH=./libcsp/build gdb python3",
```

Comment:
are you missing a python script?

Reply:
it appears that way. I actually am having trouble getting gdb to work again, and I don't see it as necessary here, so I'm removing this line from the package.